fe/fe-core/src/test/java/org/apache/doris/statistics/StatsDeriveResultTest.java

```diff
@@ -37,7 +37,7 @@ public void testUpdateRowCountByLimit() {
         Assertions.assertEquals(0, resColStats.ndv);
         Assertions.assertEquals(1, resColStats.avgSizeByte);
         Assertions.assertEquals(0, resColStats.numNulls);
-        Assertions.assertEquals(0, resColStats.dataSize);
+        Assertions.assertEquals(1, resColStats.dataSize);
```

Comment:
why case result changed?

Reply:
Because I changed the build method of ColumnStatisticsBuilder
```java   
 public ColumnStatistic build() {
        dataSize = Math.max((count - numNulls + 1) * avgSizeByte, 0);
    }
```